mmeds/lefse.py

```diff
@@ -0,0 +1,89 @@
+from mmeds.config import DATABASE_DIR
+from mmeds.tool import Tool
+
+
+class Lefse(Tool):
+    """ A class for LEfSe analysis of uploaded studies. """
+
+    def __init__(self, owner, access_code, tool_type, analysis_type,  config, testing,
+                 analysis=True, restart_stage=0, kill_stage=0, child=False):
+        super().__init__(owner, access_code, tool_type, analysis_type, config, testing,
+                         analysis=analysis, restart_stage=restart_stage, child=child)
+        load = 'module use {}/.modules/modulefiles; module load lefse;'.format(DATABASE_DIR.parent)
+        self.jobtext.append(load)
+        self.jobtext.append('{}={};'.format(str(self.run_dir).replace('$', ''), self.path))
+        self.module = load
+
+        self.subclass = 'subclass' in self.doc.reads_type
+        self.subjects = 'subjects' in self.doc.reads_type
+
+    def format_input(self):
+        """Convert uploaded .txt file into file type usable by LEfSe"""
```

Comment:
Spaces around the docstrings 
```suggestion
        """ Convert uploaded .txt file into file type usable by LEfSe """
```